### ISSUE TITLE
bug(server): bar char data incorrect

### ISSUE THREAD

**Issue by CoreyCole** [CoreyCole/HypDB#28]
Right now, the ATE only returns the bar chart data for the most fine-grained chart. If I query with grouping attribute `carrier` I get an array of bar chart data to display. This should really be a 2D array of length 1, where the 0th element is the array I'm getting.

If I query with grouping attributes `carrier` and `origin` I should get a 2D array of length 2, where the 0th element is the array I got with only carrier as the grouping attribute, and the 1st element is the more fine-grained ATE (the array we currently get when calling with 2 grouping attributes).

In summary, we need to return the ATE for each grouping attribute starting with the first. This will always be a 2D array with length >= 1

**Comment by CoreyCole** [comment]
fixed in #29
